tutorial.md

```diff
@@ -216,15 +202,6 @@ Output:
 Building API
 Building reference CLI
 Building plugins
-   flannel
```

Comment:
This should not be removed, as we now only need to build the plugins, not the library itself.